homeassistant/components/prusalink/button.py

```diff
@@ -114,7 +121,9 @@ def available(self) -> bool:
     async def async_press(self) -> None:
         """Press the button."""
         try:
-            await self.entity_description.press_fn(self.coordinator.api)
+            await self.entity_description.press_fn(
+                self.coordinator.api, self.coordinator.data
+            )
```

Comment:
```suggestion
        job_id = self.coordinator.data["job"]["id"]
        func = self.entity_description.press_fn(self.coordinator.api)

        try:
            await func(job_id)
```

Reply:
Applied manually due to "Applying suggestions on deleted lines is currently not supported".
Thanks!